index.html

```diff
@@ -84,7 +83,163 @@ <h1 class="primary py-2 my-2">
       </div>
     </section>
 
-    <
+    <section class="grid text-center m-4 p-2 projects">
+      <h1 class="primary py-2 my-4 text-center">My Recent Work</h1>
+
+      <div class="card my-2">
+        <div class="card-body d-flex flex-column justify-content-center">
+          <p class="placeholder-glow">
+            <span
+              class="placeholder col-12 bg-primary project-sample-img"
+            ></span>
+          </p>
+
+          <h5 class="card-title">Datafy</h5>
+          <ul>
+            <li class="badge text-bg-secondary">REACT</li>
+            <li class="badge text-bg-secondary">Big Query</li>
+            <li class="badge text-bg-secondary">Shopify API</li>
+            <li class="badge text-bg-secondary">
+              Google Cloud Platform Services
+            </li>
```

Comment:
- [ ] KIndly make your tech stack have the background color as the one on the provided Figma design. :wink:

   - ### Figma design
    ![image](https://user-images.githubusercontent.com/87197412/200127264-abf776d4-e7a2-4f15-aa7e-d213341ffbe1.png)

   - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/200127277-34554db6-53cf-453a-ab71-e9f13b9ed00c.png)

Reply:
Fixed
![image](https://user-images.githubusercontent.com/44822765/200138478-2eb45354-6483-41f2-8b5c-3a4956b54096.png)